src/anastomosis.jl

```diff
@@ -46,19 +46,20 @@ end
 Return the Jacobian for anastomosis equations.
 """
 function calculateJacobianAnastomosis(v1 :: Vessel, v2 :: Vessel, v3 :: Vessel, U, k)
+    @fastmath @inbounds g1 = sqrt(1-v1.Ac/U[4]^4)	# MODIFIED THIS LINE
+    @fastmath @inbounds g2 = sqrt(1-v2.Ac/U[5]^4)	# MODIFIED THIS LINE
+    @fastmath @inbounds g3 = sqrt(1-v3.Ac/U[6]^4)	# MODIFIED THIS LINE
     @fastmath @inbounds U43 = U[4]*U[4]*U[4]
     @fastmath @inbounds U53 = U[5]*U[5]*U[5]
     @fastmath @inbounds U63 = U[6]*U[6]*U[6]
 
-    @fastmath @inbounds J14 =  4.0*k[1]
+    @fastmath @inbounds J14 =  4.0*k[1]*(v1.Ac/(2*U[4]*g1) + g1)	# MODIFIED THIS LINE
+    @fastmath @inbounds J25 =  4.0*k[2]*(v2.Ac/(2*U[5]*g2) + g2) 	# MODIFIED THIS LINE
+    @fastmath @inbounds J36 = -4.0*k[3]*(v3.Ac/(2*U[6]*g3) + g3)	# MODIFIED THIS LINE
 
-    @fastmath @inbounds J25 =  4.0*k[2]
-
-    @fastmath @inbounds J36 = -4.0*k[3]
-
-    @fastmath @inbounds J41 =  U[4]*U43
-    @fastmath @inbounds J42 =  U[5]*U53
-    @fastmath @inbounds J43 = -U[6]*U63
+    @fastmath @inbounds J41 =  U[4]*U43 - v1.Ac				# MODIFIED THIS LINE
```

Comment:
e dato che qui si ricalcola `U[4]*U43`, conviene definire una variabile `U44` per questo valore in modo da calcolarla una volta sola